packages/driver/package.json

```diff
@@ -39,6 +39,7 @@
     "jquery": "3.1.1",
     "jquery.scrollto": "2.1.2",
     "js-cookie": "2.2.0",
+    "js-yaml": "3.13.1",
```

Comment:
won't this need to be a `dependency` and not just a `devDependency`?

Reply:
All dependencies in the driver (and runner and reporter) are dev dependencies because they get bundled into a single javascript file and don't need to be included in `node_modules` in the Cypress binary.